samples/src/main/java/com/example/bigquery/QueryClusteredTable.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigquery;
+
+// [START bigquery_query_clustered_table]
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.QueryJobConfiguration;
+import com.google.cloud.bigquery.TableResult;
+
+public class QueryClusteredTable {
+
+  public static void runQueryClusteredTable() throws Exception {
+    // TODO(developer): Replace these variables before running the sample.
+    String query =
+        "SELECT word, word_count\n"
+            // TODO: Replace below with your partitioned and clustered table:
+            // projectId.datasetName.tableName
```

Comment:
This should be a provided argument - all TODO's should be in the no-args function.